src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+function convertToNumber(salary) {
+  const cleanedString = salary.replace(/[$,]/g, '');
+  const number = +cleanedString;
```

Comment:
Please avoid using implicit coertion of types bcs it's hard to read, use rather `Number()` constructor function to convert it to number type

```suggestion
  const number = Number(cleanedString);
```

Reply:
oh sure :) thanks for your comments!